realm/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -30,58 +33,52 @@ public LinkView(Context context, Table parent, long columnIndexInParent, long na
         this.nativeLinkViewPtr = nativeLinkViewPtr;
     }
 
-    protected static native void nativeClose(long nativeLinkViewPtr);
```

Comment:
I was planing to update the other files as well but didn't wanted to do it as part of this PR. I will create a separate PR for the other files. 

Also there is multiple reasons for changing it:

1) IMO It becomes easier to scan a class when scrolling through it as flow isn't interrupted by one-liners.
2) While there normally is merit in keeping private methods close to where they are used, the native method declaration does not give any additional detail that can be used to reason about their behaviour. 
2) It becomes easier to compare c++ files with Java files.
3) Grouping them is consistent with the standard Java classes and every other framework I have seen that uses native code.